Library/Formula/google-perftools.rb

```diff
@@ -18,7 +16,7 @@ class GooglePerftools < Formula
   end
 
   def install
-    ENV.append_to_cflags '-D_XOPEN_SOURCE'
+    ENV.append_to_cflags "-D_XOPEN_SOURCE"
```

Comment:
If you're tweaking strings mind running `brew audit --strict google-perftools` and fixing up the other issues (e.g. a test would be amazing). Thanks so much :heart:!